src/js/tracks/text-track.js

```diff
@@ -336,7 +336,17 @@ class TextTrack extends Track {
    * @param {TextTrack~Cue} cue
    *        The cue to add to our internal list
    */
-  addCue(cue) {
+  addCue(_cue) {
+    let cue = _cue;
```

Comment:
We could. I just thought it would be better to leave the rest of the code as is and keep the change relatively self-contained.